test/CP/core/search/ilds.jl

```diff
@@ -284,11 +301,12 @@ using DataStructures
             ),
             trajectory = RL.CircularArraySLARTTrajectory(
                 capacity = 500,
-                state = Matrix{Float32} => (11, 17, 1),
+                state = SeaPearl.DefaultTrajectoryState[] => (),
```

Comment:
Why should we use a 1-element `Vector{SeaPearl.DefaultTrajectoryState}` as element of the `CircularArrayBuffer ` instead of a `SeaPearl.DefaultTrajectoryState` ?

Reply:
It's an empty `Vector{SeaPearl.DefaultTrajectoryState}`, and it's because of RL.jl. RL.jl is used to get a matrix-like state, and thus the typing in of RL.jl relies on the function `eltype(state)`, which isn't currently defined for the `DefaultTrajectoryState`.

Another solution would be to define `eltype(DefaultTrajectoryState)`, but it would be confusing in other contexts.